lib/logos.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+const Joi = require('joi')
```

Comment:
This will want updating to `const Joi = require('@hapi/joi')` in the context of #3505 I'll sort it out in one or the other depending on which order these get merged in.